packages/next/src/build/webpack-build.ts

```diff
@@ -151,6 +152,9 @@ async function webpackBuildImpl(): Promise<{
   webpackBuildStart = process.hrtime()
 
   // We run client and server compilation separately to optimize for memory usage
+  if (debugOutput) {
+    console.log('starting webpack compilation')
```

Comment:
do we want use a helper to wrap this instead of call console.log directly since it's kinda easy to add/remove by accidently. a util might be helpful for searching

Reply:
Sounds good, gonna close this in favor of a follow-up PR which will use `debug()`

Reply:
Continued in https://github.com/vercel/next.js/pull/46666